deploy/role.yaml

```diff
@@ -47,13 +47,6 @@ rules:
   - get
   - create
   - delete
-- apiGroups:
-  - tekton.dev
-  resources:
-  - '*'
```

Comment:
we need this so that we can create permissions in tekton pipelines release.yaml to watch pipelines, pipelineruns etc
```
...
\"system:serviceaccounts:openshift-pipelines-operator\" \"system:authenticated\"]) is attemptin$
 to grant RBAC permissions not currently held:\n{APIGroups:[\"tekton.dev\"], Resources:[\"clustertasks\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\$
 \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"clustertasks/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[$
"tekton.dev\"], Resources:[\"pipelineresources\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\$
pipelineresources/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"pipelineruns\"], Verb$
:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"pipelineruns/finalizers\"], Verbs:[\"get\" \"list\" \"c$
eate\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"pipelineruns/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete$
" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"pipelines\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:$
\"tekton.dev\"], Resources:[\"pipelines/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\$
taskruns\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"taskruns/finalizers\"], Verbs:[\"get\$
 \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"taskruns/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\$
 \"delete\" \"patch\" \"watch\"]}\n{APIGroups:[\"tekton.dev\"], Resources:[\"tasks\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}\n{APIG$
oups:[\"tekton.dev\"], Resources:[\"tasks/status\"], Verbs:[\"get\" \"list\" \"create\" \"update\" \"delete\" \"patch\" \"watch\"]}"}
```

Reply:
i will handle this with OLM patch

Reply:
i ll make it
```
- apiGroups:
          - tekton.dev
          resources:
          - '*'
          verbs:
          - '*'
```